lacci/lib/shoes/drawables/list_box.rb

```diff
@@ -11,9 +11,11 @@ def initialize(args = {}, &block)
       @choose = args[:choose]
 
       @selected_item = args[:selected_item]
+      @text = args[:selected_item]
```

Comment:
I don't think old Shoes had a "text" property for list boxes. What is the intent for this? Right now I can't tell if it's doing anything...

Reply:
https://github.com/scarpe-team/scarpe/blob/47cc478eb009596ce3652927b287fe43263417f4/examples/list_box-shape-report.rb#L10

its basically same as selected_item

Reply:
Ah, I'm wrong here. Shoes3 totally has text, and it's *instead* of selected_item. Will fix.